src/constants/experiment-one.js

```diff
@@ -0,0 +1,5 @@
+/* @flow */
+
+export const EXPERIMENT_ONE = {
```

Comment:
Can you change all instances of EXPERIMENT ONE to the new class name?

Reply:
sure